src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -18,16 +17,16 @@ namespace Neo.SmartContract.Native
     public abstract class NativeContract
     {
         private static readonly List<NativeContract> contractsList = new List<NativeContract>();
-        private static readonly Dictionary<UInt160, NativeContract> contractsDictionary = new Dictionary<UInt160, NativeContract>();
+        private static readonly Dictionary<string, NativeContract> contractsNameDictionary = new Dictionary<string, NativeContract>();
+        private static readonly Dictionary<UInt160, NativeContract> contractsHashDictionary = new Dictionary<UInt160, NativeContract>();
```

Comment:
If the goal of this PR it's to call native contracts in a different way, why we need a Hash for it?

Reply:
The way we call native contracts is not changed. This PR is to decouple native contracts from SYSCALL. Before this PR, we have to create a new SYSCALL for each native contract.